tools/cli/app.go

```diff
@@ -25,7 +25,7 @@ import "github.com/urfave/cli"
 const (
 	// Version is the controlled version string. It should be updated every time
 	// before we release a new version.
-	Version = "0.5.11"
+	Version = "0.6.0"
```

Comment:
Let's pending on this, I have another CLI which I want to be included in next release

Reply:
https://github.com/uber/cadence/pull/1732

Reply:
Sure